src/libs/blueprint/conduit_blueprint_mesh_examples_generate.cpp

```diff
@@ -0,0 +1,642 @@
+}
+
+//---------------------------------------------------------------------------//
+float64
+fetch_float64(const Node &opts,
+              const std::string &example_name,
+              const std::string &child_name,
+              float64 def_value)
+{
+    if(!opts.has_child(child_name))
+    {
+        return def_value;
+    }
+        
+    if(!opts[child_name].dtype().is_number())
+    {
+        CONDUIT_ERROR("example: `" << example_name
+                      << "` expects options to contain a `float64` named `"
+                      << child_name << "`");
+    }
+    return opts[child_name].to_float64();
+}
+
+//-----------------------
+}
+//-----------------------
+// end detail
+//-----------------------
+
+//---------------------------------------------------------------------------//
+void
+generate(const std::string &example_name,
+         Node &res)
+{
+    Node opts;
+    generate_default_options(example_name,opts);
+    generate(example_name,opts,res);
+}
+
+//---------------------------------------------------------------------------//
+void
+generate(const std::string &example_name,
+         const Node &opts,
+         Node &res)
+{
+    res.reset();
+
+    if(example_name == "braid")
+    {
+        std::string      mesh_type = detail::fetch_string(opts,example_name,"mesh_type");
+        conduit::index_t nx        = detail::fetch_index_t(opts,example_name,"nx");
+        conduit::index_t ny        = detail::fetch_index_t(opts,example_name,"ny");
+        conduit::index_t nz        = detail::fetch_index_t(opts,example_name,"nz",0); // optional
+
+        // wrap
+        // void braid(const std::string &mesh_type,
+        //            conduit::index_t nx,
+        //            conduit::index_t ny,
+        //            conduit::index_t nz,
+        //            conduit::Node &res);
+        braid(mesh_type,nx,ny,nz,res);
+    }
+    else if(example_name == "basic")
+    {
+        std::string      mesh_type = detail::fetch_string(opts,example_name,"mesh_type");
+        conduit::index_t nx        = detail::fetch_index_t(opts,example_name,"nx");
+        conduit::index_t ny        = detail::fetch_index_t(opts,example_name,"ny");
+        conduit::index_t nz        = detail::fetch_index_t(opts,example_name,"nz",0); // optional
+
+        // wrap
+        // void basic(const std::string &mesh_type,
+        //            conduit::index_t nx,
+        //            conduit::index_t ny,
+        //            conduit::index_t nz,
+        //            conduit::Node &res);
+        basic(mesh_type,nx,ny,nz,res);
+    }
+    else if(example_name == "strided_structured")
+    {
+        conduit::index_t    npts_x = detail::fetch_index_t(opts,example_name,"npts_x");
+        conduit::index_t    npts_y = detail::fetch_index_t(opts,example_name,"npts_y");
+        conduit::index_t    npts_z = detail::fetch_index_t(opts,example_name,"npts_z",0); // optional
+
+        // desc is an output?
+        conduit::Node desc;
```

Comment:
I don't think it is an output. Looking at the tests it looks like it is an optional input. Might want to consider how users could provide `desc` for this case when using this generate infrastructure.